src/components/work/Alert/Alert.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { Alert as ReactStrapAlert } from 'reactstrap';
+
+export interface AlertProperties {
+  msg: string,
+  color: string,
+  show: boolean,
+}
+
+const Alert = (props: AlertProperties) => (
+  <ReactStrapAlert {...props} />
+);
+
+export default Alert;
```

Comment:
this was mainly created for a central place to declare the AlertProperties interface used in several places.